### PR TITLE
Unit Tests: Improve styling of TestServer

### PR DESCRIPTION
**Pull request by patrickdupuis**
Added white spaces, line breaks, and indentations. The diff is huge, so it may be hard to review the changes on GH. The test runs OK on my system. 

I also added a `tearDown {}` which quits the server when the test is finished. Previously, this test would leave the server running when finished. This was the main motivation for improving on this test.

@telephon please have a look!

### CODE REVIEW

**Comment by patrickdupuis** [comment]
going to break those long lines into 120 chars segments, as per @brianlheim 's sugestion on slack.

File: testsuite/classlibrary/TestServer.sc [hunk 1]
```diff
@@ -38,14 +33,1297 @@ TestServer : UnitTest {
 		this.sync;
```
Comment (telephon) [inline testsuite/classlibrary/TestServer.sc]: while you are at it, could you also remove the trailing semicolon here?

File: testsuite/classlibrary/TestServer.sc [hunk 2]
```diff
@@ -38,14 +33,1297 @@ TestServer : UnitTest {
...
 	}
 
+	tearDown {
+		server.quit;
```
Comment (telephon) [inline testsuite/classlibrary/TestServer.sc]: … and here.

File: testsuite/classlibrary/TestServer.sc (hunk 1 above)
Comment (brianlheim) [reply testsuite/classlibrary/TestServer.sc]: https://github.com/supercollider/supercollider/wiki/%5BWIP%5D-Code-style-guidelines#recommendation-dont-place-a-semicolon-after-the-final-statement-of-a-method-or-function

I think we agreed this is an optional style point. any particular reason to insist here?

File: testsuite/classlibrary/TestServer.sc (hunk 2 above)
Comment (patrickdupuis) [reply testsuite/classlibrary/TestServer.sc]: I'll remove this one, and all the ones that are after calls to `this.sync`.

**Comment by patrickdupuis** [comment]
@telephon done!

**Comment by patrickdupuis** [comment]
@telephon please approve these changes when you get the chance :D

**Pull request merged**